pkg/apis/kops/cluster.go

```diff
@@ -143,13 +143,21 @@ type ClusterSpec struct {
 	// This API component is under construction, will remove this comment
 	// once this API is fully functional.
 	Assets *Assets `json:"assets,omitempty"`
+
+	// IAM field adds control over the IAM security policies applied to resources
+	IAM *IAMSpec `json:"iam,omitempty"`
 }
 
 type Assets struct {
 	ContainerRegistry *string `json:"containerRegistry,omitempty"`
 	FileRepository    *string `json:"fileRepository,omitempty"`
 }
 
+// IAMSpec adds control over the IAM security policies applied to resources
+type IAMSpec struct {
+	Legacy bool `json:"legacy"`
```

Comment:
I specified `omitempty` initially to have consistency with the other API fields. I was unsure if it may cause some problems with having the cluster creation default to a legacy value of false (not present in spec), and then the apimachinery defaults check for a nil value and set legacy to true. I should have done more validation before pushing up! :D